stdlib/public/core/UnmanagedOpaqueString.swift

```diff
@@ -340,6 +340,7 @@ extension _UnmanagedOpaqueString : _StringVariant {
   }
 
   @_versioned // FIXME(sil-serialize-all)
+  @_fixed_layout
```

Comment:
Please leave the FIXME on any fixed-layouts you add. Only the stdlib team should be able to add them without FIXMEs.

Reply:
Done